components/drata/sources/audit-hub/audit-hub.mjs

```diff
@@ -0,0 +1,28 @@
+import drata from "../../drata.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+const docsLink = "";
+
+export default {
+  key: "drata-audit-hub",
+  name: "Audit Hub",
+  description: `Emit new event for a audit hub result. [See docs here.](${docsLink})`,
```

Comment:
```suggestion
  description: `Emit a new event for an audit hub result. [See docs here.](${docsLink})`,
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's Developer docs style guide (use articles).